### DISCUSSION TITLE
What are the Start and End time for?

### DISCUSSION THREAD

**Discussion by gastonfama** [cpvalente/ontime#490] [General]
I am asking this just because I am not sure what is it used for, since I can change the order or even get an empty space between events like this.

![image](https://github.com/cpvalente/ontime/assets/73205187/252b1747-ab34-4b7b-b609-870d7e38ed2f)

Should the system auto set the start and end time based on the other events and the duration of each event and the position of the events?

Actually there can be overlapping events and you wont even notice, like this

![image](https://github.com/cpvalente/ontime/assets/73205187/e21110e4-0cff-4be4-8d94-da622ae5ead8)

Hope there is a good explination, I really like the interface, the timer and would like to start using it.

Thanks, regards.

**Comment by cpvalente** [comment]
Hi @gastonfama,

Thank you for your question.

The short answer: 
Ontime does not enforce a workflow for you, if all you want is some timers, ignore the start and end time fields

The slightly longer answer:

### What are start and end times for?

There are many ways of using the app. if you just need to set some timers, the start and end fields don't do anything for you.
However you might have a day schedule that you want to follow, the idea is that you could create timers against those events

This will allow you to get some extra benefits from the app
- You can share the schedule data with your team or use it in local signage, see for example the backstage view https://ontime.gitbook.io/v2/views/backstage-info
- You can have the timers be fully or partially automated, see roll mode https://ontime.gitbook.io/v2/features/roll
- You can keep track of the running order against the schedule and plan for running delays, the documentation here is outdated in style, but the same principles apply https://ontime.gitbook.io/v2/features/the-delay-workflow
- you can have specific views that countdown to specific events, see countdown screen https://ontime.gitbook.io/v2/views/countdown

The gist of it, is that by adding extra data to your rundown, you can leverage ontime to have a bigger role in your production workflow. What that role is would highly depend on your industry and the other pieces
Or you can ignore all of it, nothing to loose apart from the annoyance of seeing fields which have no meaning to you

### Why does ontime not fix your gaps / collisions

Again, the short answer is: we don't know what users are doing. In fact, is one of the app missions not to enforce a workflow

1. Only timers
Many users only want a bunch of timers, in which case only the duration matters. 
If we enforced a cohesive schedule the user would need to fake a schedule to be able to have the timers

2. Only schedules
There are also users who just want to use this as a signage system and don't care too much for the timer itself. In some scenarios it is perfectably acceptable that there are tasks with overlapping schedules

3. Automated systems
Several ontime users are leveraging the app for venue management. 
The rundown could look like
```
08-10 -> Venue prep
10-12 -> Band setup
12-14 -> Lunch
14-15 -> Sound check
15-17 -> Support setup
18-19 -> Dinner
19-20 -> Support band
20-21 -> Band
21-23 -> De-rig
```

With this, you can use Ontime in Roll mode to have signage in the venues which tells the different departments what is happening and when. Note the gap between `dinner ` and `support band`, the app does not enforce that you create a full schedule and has no reason to, during that time, no timer will be running, be the overall schedule is still in display and ontime is in some sort of waiting mode



Hope this helps answer
